lib/error.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+module.exports = () => {
+    var error;
+
+    error = {};
+    error.DUPLICATE_ARTIFACT = {
```

Comment:
Ah, yeah... Hm. It was really handy for me to be able to compare against the codes here, internally, but yeah, you're right, it doesn't make sense for the user. I may ask for your help with this.